src/Kovan-DssSpell.sol

```diff
@@ -33,110 +41,98 @@ contract DssSpellAction is DssAction {
-        for (uint256 i = 0; i < ilks.length; i++) {
-            bytes32 ilk = ilks[i];
-            if (IlkRegistryAbstract(ILK_REGISTRY).class(ilk) < 3) {
-                address xlip = IlkRegistryAbstract(ILK_REGISTRY).xlip(ilk);
-                DssExecLib.authorize(xlip, MCD_ESM);
-                DssExecLib.deauthorize(xlip, MCD_ESM_OLD);
-            }
-        }
-
-        DssExecLib.deauthorize(MCD_FLIP_LINK_A, MCD_ESM_OLD);
-        DssExecLib.authorize(MCD_FLIP_LINK_A, MCD_ESM);
-        
-        // ------------------  CHAINLOG  -----------------
-        
-        DssExecLib.setChangelogAddress("MCD_END", MCD_END);
-        DssExecLib.setChangelogAddress("MCD_ESM", MCD_ESM);
-        DssExecLib.setChangelogAddress("ILK_REGISTRY", ILK_REGISTRY);
-    }
+        address MCD_VAT = DssExecLib.vat();
+        address MCD_CAT = DssExecLib.cat();
+        address MCD_DOG = DssExecLib.getChangelogAddress("MCD_DOG");
+        address MCD_VOW = DssExecLib.vow();
+        address MCD_END = DssExecLib.end();
+        address MCD_ESM = DssExecLib.getChangelogAddress("MCD_ESM");
+        address CLIPPER_MOM = DssExecLib.getChangelogAddress("CLIPPER_MOM");
+        address ILK_REGISTRY = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+        address PIP_YFI = DssExecLib.getChangelogAddress("PIP_YFI");
+        address MCD_FLIP_YFI_A = DssExecLib.getChangelogAddress("MCD_FLIP_YFI_A");
+
+        // Set CLIP for YFI-A in the DOG
+        DssExecLib.setContract(MCD_DOG, "YFI-A", "clip", MCD_CLIP_YFI_A);
+
+        // Set VOW in the YFI-A CLIP
+        DssExecLib.setContract(MCD_CLIP_YFI_A, "vow", MCD_VOW);
+
+        // Set CALC in the YFI-A CLIP
+        DssExecLib.setContract(MCD_CLIP_YFI_A, "calc", MCD_CLIP_CALC_YFI_A);
+
+        // Authorize CLIP can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_CLIP_YFI_A);
+
+        // Authorize CLIP can access to DOG
+        DssExecLib.authorize(MCD_DOG, MCD_CLIP_YFI_A);
+
+        // Authorize DOG can kick auctions on CLIP
+        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_DOG);
+
+        // Authorize the new END to access the YFI CLIP
+        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_END);
+
+        // Authorize CLIPPERMOM can set the stopped flag in CLIP
+        DssExecLib.authorize(MCD_CLIP_YFI_A, CLIPPER_MOM);
+
+        // Authorize new ESM to execute in YFI-A Clipper
+        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_ESM);
+
+        // Whitelist CLIP in the YFI osm
+        DssExecLib.addReaderToOSMWhitelist(PIP_YFI, MCD_CLIP_YFI_A);
 
+        // Whitelist CLIPPER_MOM in the YFI osm
+        DssExecLib.addReaderToOSMWhitelist(PIP_YFI, CLIPPER_MOM);
+
+        // No more auctions kicked via the CAT:
+        DssExecLib.deauthorize(MCD_FLIP_YFI_A, MCD_CAT);
+
+        // No more circuit breaker for the FLIP in YFI-A:
+        DssExecLib.deauthorize(MCD_FLIP_YFI_A, DssExecLib.flipperMom());
+
+        Fileable(MCD_DOG).file("YFI-A", "hole", 5_000 * RAD);
```

Comment:
This should be 5M, maybe for kovan is fine 5k?

Reply:
Yes this is intentionally lower for kovan.